lnd_test.go

```diff
@@ -43,7 +43,11 @@ var (
 )
 
 const (
-	testFeeBase = 1e+6
+	testFeeBase         = 1e+6
+	defaultTimeout      = lntest.DefaultTimeout
```

Comment:
Other than less typing, what do we gain by re-defining these variables from the package level `lntest` variables?